### ISSUE TITLE
Nest field like STRUCT is not handled

### ISSUE THREAD

**Issue by kukigai** [xnuinside/simple-ddl-parser#137]
I tried latest version simple-ddl-parser==0.26.4 but didn't work.

Sample schema.

CREATE EXTERNAL TABLE aaa.bbb (
    test ARRAY<STRUCT<xyz: STRING COMMENT 'xxxxxx',
     date: STRING COMMENT 'xxxxxxx'>> COMMENT 'xxxxxx',
)
COMMENT 'xxxxx'
PARTITIONED BY (date STRING)
STORED AS PARQUET LOCATION 's3://xxxxx'

**Comment by xnuinside** [comment]
@kukigai  thanks for reporting the issue, I will take a look

**Comment by xnuinside** [comment]
@kukigai yeah, I don't have a time last days :/ because of work project, and not sure what will have a time till weekends

**Comment by kukigai** [comment]
Actually, it seems STRUCT is not supported at all.

**Comment by soujyosen** [comment]
Is there any update on this? This is an important use case for us with struct columns having comments.
